app/client/src/modules/ui-builder/ui/wds/WDSCustomWidget/component/index.tsx

```
@@ -1,210 +1,187 @@
-  useEffect(() => {
-    if (iframe.current && iframe.current.contentWindow && isIframeReady) {
-      iframe.current.contentWindow.postMessage(
-        {
+  useEffect(
+    // iframe can listen to changes to model with `appsmith.onModelChange` function.
+    // To do this, we send a CUSTOM_WIDGET_MODEL_CHANGE message to the iframe
+    // when the model changes. Iframe would be listening to these messages and
+    // when it receives one, it calls all the callbacks that were registered
+    // with `appsmith.onModelChange` function
+    function handleModelChange() {
+      if (iframe.current && iframe.current.contentWindow && isIframeReady) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)